x-pack/legacy/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/logstash/enable_metricbeat_instructions.js

```diff
@@ -174,60 +167,18 @@ export function getLogstashInstructionsForEnablingMetricbeat(product, _meta, {
 
   let migrationStatusStep = null;
   if (product.isInternalCollector || product.isNetNewUser) {
-    let status = null;
-    if (hasCheckedStatus) {
-      status = (
-        <Fragment>
-          <EuiSpacer size="m"/>
-          <EuiCallOut
-            size="s"
-            color="warning"
-            title={i18n.translate('xpack.monitoring.metricbeatMigration.logstashInstructions.isInternalCollectorStatusTitle', {
-              defaultMessage: `We have not detected any monitoring data coming from Metricbeat for this Logstash.
-              We will continuously check every {timePeriod} seconds in the background.`,
-              values: {
-                timePeriod: autoCheckIntervalInMs / 1000,
-              }
-            })}
-          />
-        </Fragment>
-      );
-    }
-
-    let buttonLabel;
-    if (checkingMigrationStatus) {
-      buttonLabel = i18n.translate('xpack.monitoring.metricbeatMigration.logstashInstructions.checkingStatusButtonLabel', {
-        defaultMessage: 'Checking for data...'
-      });
-    } else {
-      buttonLabel = i18n.translate('xpack.monitoring.metricbeatMigration.logstashInstructions.checkStatusButtonLabel', {
-        defaultMessage: 'Check for data'
-      });
-    }
-
     migrationStatusStep = {
       title: statusTitle,
       status: 'incomplete',
       children: (
-        <Fragment>
-          <EuiFlexGroup alignItems="center">
-            <EuiFlexItem>
-              <EuiText>
-                <p>
-                  {i18n.translate('xpack.monitoring.metricbeatMigration.logstashInstructions.statusDescription', {
-                    defaultMessage: 'Check that data is received from the Metricbeat'
-                  })}
-                </p>
-              </EuiText>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiButton onClick={checkForMigrationStatus} isDisabled={checkingMigrationStatus}>
-                {buttonLabel}
-              </EuiButton>
-            </EuiFlexItem>
-          </EuiFlexGroup>
-          {status}
-        </Fragment>
+        <EuiCallOut
+          size="s"
+          color="warning"
+          title={i18n.translate('xpack.monitoring.metricbeatMigration.logstashInstructions.isInternalCollectorStatusTitle', {
+            defaultMessage: `We have not detected any monitoring data coming from Metricbeat for this Logstash.
```

Comment:
Same here, maybe: "...for this Logstash [node]" or w/e the correct terminology is.